tests/phpunit/tests/cache.php

```diff
@@ -349,7 +349,7 @@ public function test_wp_cache_delete() {
 
 	public function test_switch_to_blog() {
 		if ( ! method_exists( $this->cache, 'switch_to_blog' ) ) {
-			$this->markTestSkipped( 'This test requires a switch_to_blog() method on the cache object.' );
+			$this->fail( 'This test requires a switch_to_blog() method on the cache object.' );
```

Comment:
Is this even a realistic scenario for a test ? I.e. how can it happen that a cache object created for this test would not have that method ? And if the method doesn't exist, wouldn't test fail no matter what, so why does the condition exist ?